src/UnwindRustSgx.c

```diff
@@ -0,0 +1,150 @@
+#define _GNU_SOURCE
+#include <link.h>
+
+#include <elf.h>
+#include <stdarg.h>
+#define max_log 256
+#include "UnwindRustSgx.h"
+
+#ifdef __cplusplus
+extern "C"  {
+#endif
+	int c_rwlock_rdlock(RWLock *rwlock);
+	int c_rwlock_wrlock(RWLock *rwlock);
+	int c_rwlock_unlock(RWLock *rwlock);
+	unsigned char *__rust_alloc(size_t, size_t);
+	void __rust_dealloc(unsigned char *, size_t);
```

Comment:
I don't think this is the correct signature for this function

Reply:
Thanks, Jethro. Will fix.